base/src/test/java/uk/ac/ebi/atlas/profiles/stream/BaselineTranscriptProfileStreamFactoryTest.java

```diff
@@ -275,6 +261,19 @@ public void rightJson(){
                                 "                \"assays\": [\"assay_3\"],\n" +
                                 "                \"value\": 3.0\n" +
                                 "            }]\n" +
+                                "        }, {\n" +
+                                "            \"stats\": {\n" +
+                                "                \"min\": 1.0,\n" +
+                                "                \"lower_quartile\": 1.0,\n" +
+                                "                \"median\": 1.0,\n" +
+                                "                \"upper_quartile\": 1.0,\n" +
+                                "                \"max\": 1.0\n" +
+                                "            },\n" +
+                                "            \"values\": [{\n" +
+                                "                \"id\": \"assay_1\",\n" +
+                                "                \"assays\": [\"assay_1\"],\n" +
+                                "                \"value\": 1.0\n" +
+                                "            }]\n" +
                                 "        }],\n" +
```

Comment:
Just a side note: I’ve been thinking about this sort of tests lately, and I think that testing with JsonPath is definitely the way to go. It will make tests more robust and easier to modify when we e.g. add fields.

Fix the test and it’s good.